packages/katello/rubygem-activerecord-import/rubygem-activerecord-import.spec

```diff
@@ -7,7 +7,7 @@
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 1.0.0
-Release: 3%{?dist}
+Release: 7%{?dist}
```

Comment:
There are newer versions on Rubygems. Can't we update instead?

Reply:
Updated the rubygem to 1.1.0